xml/System.Net.Http/HttpClient.xml

```diff
@@ -66,6 +66,8 @@ public class GoodController : ApiController
 }  
   
 ```  
+
+By default <xref:System.Net.ServicePointManager.DefaultConnectionLimit> will define the maximum number of concurrent connections to the same server. 
```

Comment:
comma after "default"
after defaultConnectionLimit, ?displayProperty=nameWithType
will define --> defines

Reply:
Thank you, i have added the changes.